packages/lib/src/components/P24/index.ts

```diff
@@ -0,0 +1,7 @@
+import IssuerListContainer from '../helpers/IssuerListContainer';
+
+class P24Element extends IssuerListContainer {
```

Comment:
Better to name the payment method to `OnlineBankingPL`, same as we do with the other online banking payment methods (`OnlineBankingCZ`, `OnlineBankingSK`..).  Also rename the folder that keeps it.